common/src/main/java/com/wynntils/features/user/overlays/InfoBoxesFeature.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user.overlays;
+
+import com.mojang.blaze3d.platform.Window;
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.config.Config;
+import com.wynntils.core.config.ConfigHolder;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.core.features.overlays.Overlay;
+import com.wynntils.core.features.overlays.OverlayPosition;
+import com.wynntils.core.features.overlays.annotations.OverlayInfo;
+import com.wynntils.core.features.overlays.sizes.GuiScaledOverlaySize;
+import com.wynntils.core.features.properties.FeatureInfo;
+import com.wynntils.mc.event.RenderEvent;
+import com.wynntils.mc.render.FontRenderer;
+import com.wynntils.mc.render.HorizontalAlignment;
+import com.wynntils.mc.render.TextRenderSetting;
+import com.wynntils.mc.render.TextRenderTask;
+import com.wynntils.mc.render.VerticalAlignment;
+import com.wynntils.utils.objects.CustomColor;
+import net.minecraft.ChatFormatting;
+
+@FeatureInfo(category = "Overlays")
+public class InfoBoxesFeature extends UserFeature {
+
+    @OverlayInfo(renderType = RenderEvent.ElementType.GUI)
+    private final Overlay infoBox1Overlay = new InfoBoxOverlay1();
+
+    @OverlayInfo(renderType = RenderEvent.ElementType.GUI)
+    private final Overlay infoBox2Overlay = new InfoBoxOverlay2();
+
+    @OverlayInfo(renderType = RenderEvent.ElementType.GUI)
+    private final Overlay infoBox3Overlay = new InfoBoxOverlay3();
+
+    @OverlayInfo(renderType = RenderEvent.ElementType.GUI)
+    private final Overlay infoBox4Overlay = new InfoBoxOverlay4();
+
+    @OverlayInfo(renderType = RenderEvent.ElementType.GUI)
+    private final Overlay infoBox5Overlay = new InfoBoxOverlay5();
+
+    @OverlayInfo(renderType = RenderEvent.ElementType.GUI)
+    private final Overlay infoBox6Overlay = new InfoBoxOverlay6();
```

Comment:
I imagined this as a special overlay that can have any amount of instances. I just did not add framework for it yet. This is fine until I do that.

Reply:
If you are doing this for the different translations, you can just overwrite the getTranslatedName to add the id of the overlay.